.github/workflows/sanger_test.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Launch workflow via tower
-        uses: seqeralabs/action-tower-launch@v2
+        uses: seqeralabs/action-tower-launch@2.0.0
```

Comment:
Can we first try without this version change ? I only ever see the `@v2` syntax, not the`@2.0.0` one

Reply:
https://github.com/marketplace/actions/action-tower-launch
The green button on the top right said using `@2.0.0`. I can't see why it doesn't work use v2, that is why I just give a try.